index.html

```diff
@@ -9,187 +9,230 @@
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#contact">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </ul>
+    </nav>
+    
     <header>
-        <nav>
-            <ul  class="navbar">
-                <li><a class="logo" href="">My Logo</a></li>
-                <li> <a href=""><img class="toggler-icon" src="img/hamburger-icon.png" alt="Hamburger Icon"></a></li>
-            </ul>
-        </nav>
+    <section class="introsection">
+        <div class="introsection-text">
+            <p>I'm Joshua</p>
+            <p>Glad to see you!</p>
+            <p id="intro-details">I’m a software developer!
+            I can help you build a product, 
+            feature or website Look 
+            through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            </div>
+
+        <div class="introsection-text" id="top">
+            <h3 id="intro-connect">LET'S CONNECT</h3>
+              <ul class="socials">
+                <li>
+                    <a href="#"><img src="img/twitter-icon.png" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="img/linkedin-icon.png" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="img/medium-icon.png" alt="medium"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="img/github-icon.png" alt="github"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="img/angellist-icon.png" alt="angellist"></a>
+                </li>
+              </ul>
+            </div>
+    </section>
     </header>
-    <main>
-    <section>
-        <h1 id="intro">I'm Joshua<br>Glad to see you!</h1>
-        <p id="intro-details">I’m a software developer!
-            I can <br> help you build a product, 
-            feature or website <br> Look 
-            through some of my work and <br> experience! 
-            If you like what you see and <br> have a project you need coded, 
-            don’t <br> hestiate to contact me.</p>
-        </section>
-    <div class="connect">
-        <h3 id="intro-connect">LET'S CONNECT</h3>
-        <ul class="socials">
-            <li><a href=""><img src="img/twitter-icon.png" alt=""></a></li>
-            <li><a href=""><img src="img/linkedin-icon.png" alt="linkedin"></a></li>
-            <li><a href=""><img src="img/medium-icon.png" alt=""></a></li>
-            <li><a href=""><img src="img/github-icon.png" alt=""></a></li>
-            <li><a href=""><img src="img/angellist-icon.png" alt=""></a></li>
-        </ul>
-    </div>
 
-    <section class="work-section">
```

Comment:
- [ ] Kindly note that the requirements for this milestone is that you use `display:grid` to style your cards and on the individual card use `display:flex` to arrange contents in the cards both in your mobile version as well as desktop version.
<img width="1680" alt="Screenshot 2022-09-30 at 04 41 47" src="https://user-images.githubusercontent.com/48546992/193246158-2d074217-2d51-4d1d-b2b8-8c18e71f0790.png">